.github/workflows/swift-test.yml

```diff
@@ -14,8 +14,9 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-      with:
-        swift-version: '5.2.0'
-
+    - name: Download and Run  MeiliSearch
+      run:  brew update && brew install meilisearch
+    - name: Run MeiliSearch
+      run: meilisearch --no-analytics=true &
```

Comment:
don't forget the masterKey 🙂

Reply:
I'm waiting for this bug to be fixed before adding it: #39